pandas/tests/test_arithmetic.py

```diff
@@ -73,6 +73,41 @@ def box_df_fail(request):
     return request.param
 
 
+# ------------------------------------------------------------------
+# Numeric types Arithmetic Operations
+
+class TestNumericOps(object):
+
+    @pytest.mark.parametrize('opname', ['neg', 'pos', 'abs', 'inv', 'invert'])
+    @pytest.mark.parametrize('dtype', [int, float, object])
+    def test_numeric_unary_ops(self, box, dtype, opname):
+        # GH 22335
+        data = [-2, 0, 4, 5]
+        obj = box(data, dtype=dtype)
+
+        def call_method():
+            return getattr(obj, '__{}__'.format(opname))()
+
+        if opname == 'inv':
+            # GH 22335 - 'inv' was wrong and was removed
+            pytest.raises(AttributeError, call_method)
```

Comment:
make these if/elseif, then don't need the return (put the final clause in the else)